### ISSUE TITLE
Add command or flag to show manifest verbatim (in CLI) - i.e. `winget cat` or `winget show --verbatim`

### ISSUE THREAD

**Issue by g0t4** [microsoft/winget-cli#1458]
# Description of the new feature/enhancement

When installing packages I like to review what will happen. Often in detail. Especially for first time packages. 

- I look for things like, are there any `http` links--instead of `https`--that could be a risk?
- Are URLs official (i.e. for downloading installers, assets, etc)? If not, why not?
- Occasionally, I look at what install options are passed, or compilation flags enabled.
    - Can I override any of them?
- Sometimes (if the information is available in the installer) I look at what exactly is changed on disk for any number of reasons.  Obviously this depends on how the installation happens and isn't always possible. But say a script is used versus a programmatic installer. 

# Proposed technical implementation details (optional)

Homebrew has `brew cat` for this very task. Hence the suggestion of `winget cat`

Or, we already have `winget show` and it dumps much of the information, but not all of it (verbatim). So, another option would be to add a `winget show --verbatim` flag or similar. In fact, this is where I first looked to see if the option already exists (hopefully I didn't miss it)!

Right now, I can poke around on GitHub and infer what manifest is what. But I can easily make a mistake. It's potentially time consuming. I could look at the wrong version. And I'd prefer to stay in the CLI flow quite often and not need a browser too.

I have some **companion** requests for similar commands to provide additional information about a package, I'll open separate requests for those so as not to put too much in here.

**Comment by denelon** [comment]
We've discussed this internally. In some cases, there are many installer nodes. I think we're leaning towards `winget show foo --verbose`.

**Comment by doctordns** [comment]
A better way to achieve this is via CMDLETs and using -WHATIF

**Comment by Swamination** [comment]
I agree this would be a nice default feature so that winget can interact meaningfully with the source manifest. There are automations and filters that may require that information specified by the creator of the manifest (custom cli args). A workaround via Powershell:
`(Invoke-WebRequest -URI "https://raw.githubusercontent.com/microsoft/winget-pkgs/master/manifests/a/Adobe/Acrobat/Reader/64-bit/21.007.20099/Adobe.Acrobat.Reader.64-bit.installer.yaml" -UseBasicParsing).Content`

**Comment by denelon** [comment]
Duplicate of #159
